functions.h

```diff
@@ -0,0 +1,288 @@
+#ifndef functions_H
+#define functions_H
+
+using json = nlohmann::json;
+
+void make_transaction(const int& id1, const int& id2, const int& money)
```

Comment:
It would be preferable to make a class for these functions, declare them in .h file and define in .cpp